btrfs_hook

```diff
@@ -68,8 +68,8 @@ BTRFS_DIR_ROLLBACK="${BTRFS_DIR_ROLLBACK%/}"
 : ${BTRFS_BOOT_DELAY:=0}
 
 root_is_btrfs() {
-    [ "$(blkid -s TYPE -o value "$root" )" != "btrfs" ] &&
-        [ "$(blkid -s TYPE -o value -lt "$root" )" != "btrfs" ]
+    [ "$(blkid -s TYPE -o value \"$root\" )" != "btrfs" ] &&
+        [ "$(blkid -s TYPE -o value -lt \"$root\" )" != "btrfs" ]
```

Comment:
How was this failing? What was the error message? This seems wrong to me -- quoting the `"` will make it part of the value.